api.py

```diff
@@ -0,0 +1,143 @@
+from configuration import DB
+import json
+
+from flask import Blueprint, request, make_response, stream_with_context
+
+from DbManager import DbManager
+
+api = Blueprint('api', __name__)
+from Lobby import Lobby_Manager
+
+class DepthResponse:
+    SUCC = "SUCCESS"
+    ERROR = "ERROR"
+
+    def __init__(self, type, message="", data=""):
+        self.type = type
+        self.message = message
+        self.data = data
+
+    def to_dict(self):
+        response = dict()
+        response['type'] = self.type
+        response['message'] = self.message
+        response['data'] = self.data
+        return response
+
+    def to_json(self):
+        return json.dumps(self.to_dict())
+
+INVALID_LOBBY_UUID_RES = (DepthResponse(DepthResponse.ERROR, message="Invalid Lobby UUID").to_json(), 400)
+
+# Lobby Support
+@api.route('/api/lobby/create')
```

Comment:
CRUD verbs will map to the HTTP verbs (GET, POST, etc.) https://stackoverflow.blog/2020/03/02/best-practices-for-rest-api-design/

so for the url it should just be /api/lobby, since a POST @ /api/lobby will create it

Reply:
that article is pog, i will update code